src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir-Book ;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy ;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.input {
+  font-family: inherit;
+  font-size: 20px;
+  font-weight: 300;
+  box-sizing: border-box;
+  width: 100%;
+  margin: 0 0 20px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.input::placeholder {
+  font-style: inherit;
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  font-family: inherit;
+}
+
+.image-big {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  bottom: 26px;
+}
+
+.image-small {
+  width: 11px;
+  height: 11px;
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.input:active {
+  font-style: inherit;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
```suggestion
.input:focus {
  font-style: inherit;
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```
`:active` is for links 😉
and you need to remove the outline 🙃

<img width="249" alt="image" src="https://user-images.githubusercontent.com/55802630/172619369-899a06c5-0118-439c-9336-ceadfc022404.png">